static/app/components/profiling/frameStack.tsx

```diff
@@ -524,13 +562,14 @@ const FrameCallersTableHeader = styled('div')`
 `;
 
 const FrameCallersTableCell = styled('div')<{
+  noPadding?: boolean;
   textAlign?: React.CSSProperties['textAlign'];
 }>`
-  width: 156px;
+  width: 164px;
```

Comment:
Not a big fan of these magic numbers, can we have a const somewhere?

Reply:
Sure, I have no strong opinion on this tbh, but since it's used in 2 places probably good to make it a const 🤷🏼